docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade to paid
```

Comment:
Need 2 changes here:

1. Change the filename to `cloud-org-upgrade-to-paid-account.md`
2. Change the YAML title to same:
```
title: Upgrade to paid account
```

This is because of the issue I highlighted in the first review: Paid **what**?